src/fsharp/FSharp.Core/local.fs

```diff
@@ -988,6 +988,12 @@ module internal List =
             takeWhileFreshConsTail cons p xs
             cons
 
+    let rec tryLast (list: 'T list) = 
```

Comment:
Call this `tryLastV` since it's returning a value option (even though it's internal let's give it the suffix naming we'd be likely to adopt elsewhere)

Reply:
done in https://github.com/dotnet/fsharp/pull/7765/commits/055124ba6b00c76137a6d5c5714e6f93b5152c49